test/api/controllers/remoteActionsController/prepareDb.test.js

```diff
@@ -306,12 +304,10 @@ describe('Test: Prepare database', function () {
       importMapping,
       workerCalled,
       mappingCreated,
-      workerPromise,
-      prepareDb;
+      workerPromise;
 
     before(function () {
-      prepareDb = rewire('../../../../lib/api/controllers/remoteActions/prepareDb');
-      importMapping = prepareDb.__get__('importMapping');
+      importMapping = rewire('../../../../lib/api/controllers/remoteActions/prepareDb').__get__('importMapping');
```

Comment:
The only thing I did is to remove the set `prepareDb` once every `before`, as it is already done in `beforeEach`, it seamed to do twice the job without any need. I think the complete test should be redone with `sinon`, but as the scope of this PR is already a bit too large, I didn't do it.